packages/plugins/users-permissions/server/controllers/role.js

```diff
@@ -30,13 +39,17 @@ module.exports = {
       return ctx.notFound();
     }
 
-    ctx.send({ role });
+    const safeRole = await sanitizeOutput(role);
```

Comment:
Can't this use the defaultSanitizers instead of manually requiring i18n? I don't see how we need this specifically here ?

Reply:
I didn't want to change the current behavior of returning all the fields (including potentially private ones) since it wasn't present before, hence the custom sanitization.

If you have more context on what should/shouldn't be returned from this API and why, I'm all ears :sweat_smile:

Reply:
I don't know, just strange to see this done here only 🤔